routes/index.js

```diff
@@ -9,61 +9,63 @@ var fs = require('file-system');
 var officegen = require('officegen');
 var passportFunction = require('../lib/passport');
 var smtp = require('../lib/smtp');
+var Language = require('../models/language')
 
-router.get('/updateschema', passportFunction.adminLoggedIn, function(req, res, next) {
-    // var songs = [{
-    //     "lyric": [
-    //         "A mighty fortress is our God,",
-    //         "A bulwark never failing;",
-    //         "Our helper He, amid the flood",
-    //         "Of mortal ills prevailing.",
-    //         "For still our ancient foe",
-    //         "Doth seek to work us woe;",
-    //         "His craft and pow'r are great,",
-    //         "And, armed with cruel hate,",
-    //         "On earth is not his equal.",
-    //         '',
-    //         "Joy to the World , the Lord is come!",
-    //         "Let earth receive her King",
-    //         "Let every heart prepare Him room",
-    //         "And Heaven and nature sing",
-    //         "And Heaven and nature sing",
-    //         "And Heaven, and Heaven, and nature sing",
-    //         "",
-    //         "Joy to the World, the Savior reigns!",
-    //         "Let men their songs employ",
-    //         "While fields and floods, rocks, hills and plains",
-    //         "Repeat the sounding joy",
-    //         "Repeat the sounding joy",
-    //         "Repeat, repeat, the sounding joy"
-    //     ],
-    //     private: false
-    // }]
-    var stanza = [];
-    var newLyric = [];
-    Song.find(function(err, songs) {
-        songs.forEach((song, i, arr) => {
-            song.lyric.forEach((line) => {
-                if (line !== '') {
-                    stanza.push(line)
-                } else {
-                    newLyric.push(stanza)
-                    stanza = [];
-                }
-            })
-            newLyric.push(stanza)
-            stanza = [];
-            song.lyric = newLyric;
-            delete song.private
-            song.save(function(err) {
-                if (err) next(err)
-                if (i === arr.length - 1) {
-                    res.send('migration success')
-                }
-            })
-        })
-    })
-})
+// router.get('/updateschema', passportFunction.adminLoggedIn, function(req, res, next) {
```

Comment:
Let's move this to a script instead of running it in the server. It should load the mongo config from a private config file or the commandline.